### PR TITLE
Replace current editor with tinymce and add media manager

### PR DESCRIPTION
**Pull request by danielsGrietins**

### CODE REVIEW

File: src/Http/Requests/AttachmentRequest.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+//            'file' => 'required|max:' . config('laradium.file_size')
```
Comment (dmitrijsmihailovs) [inline src/Http/Requests/AttachmentRequest.php]: This probably should not be commented

File: src/Http/Resources/AttachmentResource.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+     * @return array
+     */
+    public function toArray($request): array
+    {
+        return [
+            'id'         => $this->id,
+            'name'       => $this->file_file_name,
+            'url'        => $this->file->url(),
+            'is_image'   => $this->isImageMimeType($this->file_content_type),
+            'extension' => pathinfo($this->file_file_name, PATHINFO_EXTENSION)
```
Comment (dmitrijsmihailovs) [inline src/Http/Resources/AttachmentResource.php]: Formatting

File: src/Models/Attachment.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        'file'
+    ];
+
+    /**
+     * @param array $attributes
+     */
+    public function __construct(array $attributes = [])
+    {
+        $this->hasAttachedFile('file', [
+            'path'     => '/attachment/:hash/:filename',
```
Comment (dmitrijsmihailovs) [inline src/Models/Attachment.php]: Formatting

File: src/Http/Controllers/Admin/AttachmentController.php [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+     * @return JsonResponse
+     * @throws \Exception
+     */
+    public function destroy(Attachment $attachment): JsonResponse
+    {
+        // delete attachment itself
+        $attachment->file->setToBeDeleted();
+        $attachment->save();
+
+        // delete databe entry
```
Comment (dmitrijsmihailovs) [inline src/Http/Controllers/Admin/AttachmentController.php]: Comment typo. And ->delete() does not already delete attachment by default too?

File: resources/assets/js/laradium.js [hunk 5]
```diff
@@ -94,6 +80,27 @@ Vue.use(VueGoogleMaps, {
...
+import 'tinymce/plugins/autolink';
+import 'tinymce/plugins/lists';
+import 'tinymce/plugins/table';
+import 'tinymce/plugins/hr';
+import 'tinymce/plugins/fullscreen';
+import 'tinymce/plugins/preview';
+
+Vue.component('editor', require('@tinymce/tinymce-vue').default);
+
+// import Editor from '@tinymce/tinymce-vue';
```
Comment (dmitrijsmihailovs) [inline resources/assets/js/laradium.js]: This probably should be removed

File: src/Http/Controllers/Admin/AttachmentController.php (hunk 4 above)
Comment (danielsGrietins) [reply src/Http/Controllers/Admin/AttachmentController.php]: No, it only deletes database entries and leaves attachments untouched

**Pull request merged**